src/main/java/ca/on/oicr/gsi/dimsum/CaseLoader.java

```diff
@@ -713,22 +713,23 @@ private <T> List<T> loadFromJsonArrayFile(FileReader fileReader, ParseFunction<J
     return loaded;
   }
 
-  public static Map<String, ProjectSummary> calculateProjectSummaries(List<Case> cases) {
+  public static Map<String, ProjectSummary> calculateProjectSummaries(List<Case> caseList,
+      Map<Case, List<Test>> map) {
```

Comment:
One of these parameters will/should always be null, so a cleaner solution would be to make two separate methods. You could still share most of the code by separating all of the counting into an `addCounts(Case, List<Test>, Map<String, ProjectSummary.Builder>)` method. The bit at the end could also be moved into a buildProjectSummaries(Map<String, ProjectSummary.Builder>)` method to share

Reply:
@djcooke Do you think it would be better to add both methods to `caseloader`? Or one of them on `caseService`. Only `caseService` is using the filtered version, but I am wondering if it would be better style to put all of them together to minimize confusion in the future

Reply:
Both in `CaseLoader` is ideal so all of that logic is in one place